action.yml

```
@@ -29,3 +29,6 @@ inputs:
   validateSingleCommit:
     description: "When using \"Squash and merge\" on a PR with only one commit, GitHub will suggest using that commit message instead of the PR title for the merge commit, and it's easy to commit this by mistake. Enable this option to also validate the commit message for one commit PRs." 
     required: false
+  githubServerUrl:
+    description: "Github Enterprise server URL. Set this if using GHES. E.g. https://github.myorg.com/api/v3"
```

What do you think about using `githubBaseUrl` as the name to stick to the naming convention from the octokit client?

```suggestion
  githubBaseUrl:
    description: "If you use Github Enterprise, you can set this to the URL of your server (e.g. https://github.myorg.com/api/v3)"
```

Can you please also document this new parameter in the README?

---

renamed to `githubBaseUrl`. thanks for the suggestion!

also added it to the README